pkg/core/storage/boltdb_store.go

```diff
@@ -21,6 +22,10 @@ type BoltDBStore struct {
 	db *bbolt.DB
 }
 
+// defaultOpenTimeout is the default timeout for performing flock on a bbolt database.
+// bbolt does retries every 50ms during this interval.
+const defaultOpenTimeout = 5 * time.Second
```

Comment:
We've recently changed NeoFS to use 1s (nspcc-dev/neofs-node#2499), likely it fits well here as well.

Reply:
Fixed